src/modules/shortcut_guide/shortcut_guide.h

```diff
@@ -68,4 +68,16 @@ class OverlayWindow : public PowertoyModuleIface
             { L"dark", IDS_SETTING_DESCRIPTION_THEME_DARK }
         };
     } theme;
+
+    struct DisplayOnMethod
+    {
+        PCWSTR name = L"display on method";

```

Comment:
This string and also the other strings at line 75, 78, 79 and 80, are used as IDs, so it's better to not use whitespace but the undersore, that makes it clear they are strings that should not be localized.